src/core/server/opensearch/opensearch_config.ts

```diff
@@ -131,6 +131,7 @@ export const configSchema = schema.object({
   healthCheck: schema.object({ delay: schema.duration({ defaultValue: 2500 }) }),
   ignoreVersionMismatch: schema.boolean({ defaultValue: false }),
   disablePrototypePoisoningProtection: schema.maybe(schema.boolean({ defaultValue: false })),
+  compression: schema.maybe(schema.boolean({ defaultValue: false })),
```

Comment:
Why do we default this to false? Why would i not want this by default?

Reply:
There could be a little bit of overhead. Without much insights on performance, likely this is more safe set to false.

Reply:
+1 to @kavilla we should be more conservative before we have performance and load test as part of CI